src/bin/tikv-server.rs

```diff
@@ -198,6 +198,14 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
     };
     opts.set_min_write_buffer_number_to_merge(min_write_buffer_number_to_merge as i32);
 
+    let max_background_threads = get_integer_value("",
```

Comment:
@huachaohuang thanks for your comment, but that is not exactly right. let's see rocksdb's code.

db/db_impl.cc
`DBImpl::DBImpl(const DBOptions& options, const std::string& dbname)
    : env_(options.env),
      dbname_(dbname),
      db_options_(**SanitizeOptions**(dbname, options)),
      stats_(db_options_.statistics.get()),

SanitizeOptions(...) { 
 result.env->IncBackgroundThreadsIfNeeded(src.**max_background_compactions**,Env::Priority::LOW);
  result.env->IncBackgroundThreadsIfNeeded(src.**max_background_flushes**,Env::Priority::HIGH);
}`
As we can see, max_backgroud_compactions can affect the ENV variable.